src/netgroup.h

```diff
@@ -5,9 +5,41 @@
 #ifndef BITCOIN_NETGROUP_H
 #define BITCOIN_NETGROUP_H
 
+#include <vector>
+
 /**
  * Netgroup manager
  */
-class NetGroupManager {};
+class NetGroupManager {
```

Comment:
I feel like I am missing a bit of context on the naming decision of this class. The name is pretty generic so is the goal to add more functionality not related to ASMap? If not I would have probably opted for `AsmapMan` or so. If there is more to add here in the future maybe it would be good to add some more context in the comment above. Or maybe NetGroup is already a term used in this context and I am not aware of this?